packages/types/product.ts

```diff
@@ -1,13 +1,15 @@
 import { z } from "zod";
 import { ZEnvironment } from "./environment";
 import { ZColor, ZPlacement } from "./common";
+import { ZTeamBilling } from "./teams";
 
 export const ZProduct = z.object({
   id: z.string().cuid2(),
   createdAt: z.date(),
   updatedAt: z.date(),
   name: z.string(),
   teamId: z.string(),
+  billingInfo: ZTeamBilling.optional(),
```

Comment:
please remove this from the product service as we can easily get this from the team service and we don't want to query an additional table every time we request a product.